packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -1,7 +1,13 @@
 import { attr, FASTElement } from '@microsoft/fast-element';
 import * as d3 from 'd3';
+import { createTabster, getGroupper, getMover, getTabsterAttribute, Types } from 'tabster';
 import { IChartDataPoint, IChartProps, Variant } from './horizontalbarchart.options.js';
 
+// During the page startup.
```

Comment:
this PR depends on the tabster PR.

Reply:
Ohh yes, will remove tabster from here then

Reply:
Or Instead we can keep it here, anyways that PR will also get merged.